Shapes.java

```diff
@@ -1,25 +1,32 @@
 import java.util.*;
+import java.lang.Math;
 
-public class Shapes{
-  public static void main(String[] args) {
-    Scanner input = new Scanner(System.in);
-    System.out.print("Enter the side length: ");
-    double side = input.nextDouble();
-    square(side);
-  }
-
-  /*
-   * N: square
-   * P: to calculate the area of any square
-   * I: a side measure (double)
-   * R:
-   */
-  public static void square(double s){
-    //side times side
-    double a = s * s;
+public class Main
+{
+	public static void main(String[] args) {
+    square(5);
+    trangle(6,6);
+		trangle(4.0,5.0,6.0);
+}
+public static void square(double s) {
+double a = s * s;
+System.out.println(a);
+}
+public static void trangle(double b,double h) {
```

Comment:
I like the variable names you've used here. They're concise but they get the message across of what they represent.